desktop/linux/release-notes/index.md

```diff
@@ -10,6 +10,21 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.8.2
+2022-05-17
+
+> Download Docker Desktop
+>
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) |
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64) |
+> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64)
+
+### Bug fixes and minor changes
+
+- Fixed an issue with manual proxy settings which caused problems when pulling images. Fixes [docker/for-win#12714](https://github.com/docker/for-win/issues/12714) and [docker/for-mac#6315](https://github.com/docker/for-mac/issues/6315).
+- Fixed high CPU usage when extensions are disabled. Fixes [docker/for-win#12266](https://github.com/docker/for-win/issues/12266) and [docker/for-mac#6310](https://github.com/docker/for-mac/issues/6310).
```

Comment:
It doesn’t really fix the for-win issue linked here (which was open in October 2021), only partly, because this issue is really a collection of CPU hog issues.